cmd/client/wallet/main.go

```diff
@@ -767,11 +767,12 @@ func processTransferCommand() {
 
 	fmt.Printf("Unlock account succeeded! '%v'\n", senderPass)
 
-	chainConfig := params.MainnetChainConfig
-	if walletProfile.Profile != defaultProfile {
-		chainConfig = params.TestnetChainConfig
+	chainID, ok := new(big.Int).SetString(walletProfile.ChainID, 0)
```

Comment:
Can we not add a chain Id field in the profile and just infer it from the network name?

Reply:
No.  `.ini` section names are not supposed to be special.  People wanting to rename their sections should be free to do so, e.g. `pangaea` → `p`.  The ability to specify chain ID grants that freedom.